common/frame.go

```diff
@@ -737,24 +717,28 @@ func (f *Frame) ChildFrames() []api.Frame {
 func (f *Frame) Click(selector string, opts goja.Value) {
 	f.log.Debugf("Frame:Click", "fid:%s furl:%q sel:%q", f.ID(), f.URL(), selector)
 
-	parsedOpts := NewFrameClickOptions(f.defaultTimeout())
-	err := parsedOpts.Parse(f.ctx, opts)
-	if err != nil {
+	popts := NewFrameClickOptions(f.defaultTimeout())
+	if err := popts.Parse(f.ctx, opts); err != nil {
+		k6Throw(f.ctx, "%w", err)
+	}
+	if err := f.click(selector, popts); err != nil {
 		k6Throw(f.ctx, "%w", err)
```

Comment:
Not a fan of this naming tbh, it looks like it's a wrapper around panic. Can't do much about this though

Reply:
We might discuss it in a new issue if you have some other ideas? 💡

Reply:
When i saw `k6Throw`, i thought "Go doesn't have throw and catch like Java", and so I think naming it as `k6Panic` might be a little better. If someone new to Go sees this then they might get confused and expect to be able to `catch` errors... which is possible with `recover` but something we should avoid to reduce the complexity

Reply:
That makes a lot of sense. Would you like to open it up in a new discussion? :)

Reply:
Yeah sure, what's the process for starting a new discussion?

Reply:
Here: https://github.com/grafana/xk6-browser/discussions ;)

Reply:
@ankur22, I made a refactoring according to your suggestion, see: https://github.com/grafana/xk6-browser/pull/319